File: Practice/src/introduction_coding_tests/SortString1.java
package introduction_coding_tests;

public class SortString1 {
//	문자열 정렬하기 (1)
	public int[] solution(String my_string) {
		int[] answer = {};
		return my_string.chars().filter(i -> i <= '9').map(i -> i - 48).sorted().toArray();
	}
}

File: Practice/src/introduction_coding_tests/SortString1.js
function solution(my_string) {
    return [...my_string.replaceAll(/[A-Za-z]/g, '')]
        .map(Number)
        .sort((a, b) => a - b);
    // return my_string
    //     .match(/\d/g)
    //     .sort((a, b) => a - b)
    //     .map((n) => Number(n));
}
